scripts/next-data/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,53 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import { getRelativePath } from './_helpers.mjs';
+
+const generateNodeReleasesData = async () => {
+  const [nodevuOutput, indexJsonOutput] = await Promise.all([
+    nodevu(),
+    // nodevu doesn't return release date and modules
+    // So it is a temporary workaround
+    (await fetch('https://nodejs.org/dist/index.json')).json(),
+  ]);
+
+  // Filter out those without documented support
+  // Basically those not in schedule.json
+  const majors = Object.values(nodevuOutput).filter(major => {
+    return major?.support;
+  });
+
+  let i = 0;
```

Comment:
What is going on here? This is because of that hack? See, another reason why to not do this 😅